uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-calendar.component.js

```diff
@@ -95,30 +100,41 @@ define([
       vm.months = months;
     }
 
+    /**
+     * Returns a month index in the format "YYYY-<month_number>"
```

Comment:
`YYYY-MM`

Reply:
Not really. `MM` means `01`, `02`, `03` etc, however changed `number` to `index`. `Index` starts with `0`, then `1`, `2` etc.

![image](https://user-images.githubusercontent.com/3973243/40299782-4879622e-5cdf-11e8-8c77-f8e0a1465876.png)

Reply:
Done.